tests/translator/input/api_with_auth_all_minimum.yaml

```diff
@@ -32,6 +32,12 @@ Resources:
             Identity:
               Headers:
                 - Authorization1
+  MyAuthFn:
```

Comment:
Where did these new Resources come from?

Reply:
The new resources are the resources that are referenced in the template but weren't actually in the template. CFN lint does some resource reference resolution to find broken or missing resource references and dependencies. I added in what was necessary to make all of these templates valid.